src/cata_utility.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CATA_UTILITY_H
+#define CATA_UTILITY_H
+
+#include "enums.h"
+#include "creature.h"
+#include "item.h"
```

Comment:
A forward declaration of Creature should be enough for the header. Similar with item, map_item_stack and nc_color.